apps/services/handlers.py

```diff
@@ -341,7 +395,7 @@ async def __create_user_document_content_hostel_settlement_service(
             f"hostel_settlement_{file_date_created}_"
             f"{kwargs.get('user_request_id')}.docx"
         )
-        DOCUMENT_PATH = SETTLEMENT_HOSTEL_PATH / str(context.user_id)
+        DOCUMENT_PATH = SETTLEMENT_HOSTEL_PATH / str(context['user_id'])
```

Comment:
It works with dictonaries and also with 'row' variable types, where we have DB recordings